### PR TITLE
Gherkin wanleg

### PR DESCRIPTION
**Pull request by wanleg**

### CODE REVIEW

File: keyboards/gherkin/keymaps/wanleg/keymap.c [hunk 1]
```diff
@@ -14,6 +14,48 @@
...
+////////// ProMicro Bootloader Section START //////////
+//this section can be used if the "RESET" keycode doesn't work for some reason
+//shouldn't hurt anything if you don't use it, and you can delete it if you don't want it
+#include <avr/wdt.h>
+/* id for user defined functions */
+enum function_id {
+    PROMICRO_PROGRAM,
+};
+
+void promicro_bootloader_jmp(bool program) {
```
Comment (drashna) [inline keyboards/gherkin/keymaps/wanleg/keymap.c]: This code is redundant.

As long as the board has `BOOTLOADER = caterina` defined in it's rules.mk, it should be doing this with the normal reset command. 

https://github.com/qmk/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c#L135-L152

File: keyboards/gherkin/keymaps/wanleg/config.h [hunk 2]
```diff
@@ -2,6 +2,8 @@
 #define CONFIG_H
 
 #include "config_common.h"
+/*required for RESET keycode to work on ProMicro*/
+#define CATERINA_BOOTLOADER
```
Comment (drashna) [inline keyboards/gherkin/keymaps/wanleg/config.h]: If it's set up right, this define isn't needed here.

**Comment by wanleg** [comment]
made the recommended corrections - flashed & tested. looks good.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**